src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -1280,6 +1283,72 @@ def validate_custom_error_pages(namespace):
     namespace.custom_error_pages = values
 
 
+def validate_custom_headers(namespace):
+
+    if not namespace.monitor_custom_headers:
+        return
+
+    values = []
+    for item in namespace.monitor_custom_headers:
+        try:
+            item_split = item.split('=', 1)
+            values.append({'name': item_split[0], 'value': item_split[1]})
+        except IndexError:
+            raise CLIError('usage error: --custom-headers KEY=VALUE')
+
+    namespace.monitor_custom_headers = values
+
+
+def validate_status_code_ranges(namespace):
+
+    if not namespace.status_code_ranges:
+        return
+
+    values = []
+    for item in namespace.status_code_ranges:
+        item_split = item.split('-', 1)
+        usage_error = CLIError('usage error: --status-code-ranges VAL | --status-code-ranges MIN-MAX')
```

Comment:
Are MIN and MAX inclusive or exclusive? As written, I assume inclusive.

Reply:
The SDK is ambiguous on this, but yes I would assume inclusive. If you give a single value, it simply uses that value as both min and max.